src/api/java/baritone/api/Settings.java

```diff
@@ -1183,6 +1183,21 @@
      */
     public final Setting<Boolean> notificationOnMineFail = new Setting<>(true);
 
+    /**
+     * Disallow block placement and breaking within a range from the home waypoint(if it exists)
+     */
+    public final Setting<Double> homeProtectionRange = new Setting<>(0.0);
+
+    /**
+     * Sends a chat notification to let you know when you enter or leave protected home area, can be quite annoying when you're working close to a home waypoint
```

Comment:
You should split really long lines like this one into multiple lines, so they fit onto (most) screens.

Reply:
Yeah that would be better

Reply:
Changed in latest commit